Extension/src/LanguageServer/settings.ts

```diff
@@ -959,6 +960,7 @@ export class OtherSettings {
     public get filesExclude(): vscode.WorkspaceConfiguration | undefined { return vscode.workspace.getConfiguration("files", this.resource).get("exclude"); }
     public get filesAutoSaveAfterDelay(): boolean { return vscode.workspace.getConfiguration("files").get("autoSave") === "afterDelay"; }
     public get InlayHintsEnabled(): boolean { return vscode.workspace.getConfiguration("editor.inlayHints").get<string>("enabled") !== "off"; }
+    public get InlineParameterHintsEnabled(): boolean | undefined { return vscode.workspace.getConfiguration("editor.parameterHints").get<boolean>("enabled"); }
```

Comment:
Change "InlineParameterHintsEnabled" to "ParameterHintsEnabled" and other associated references.

Reply:
Should these all be camelCase?

Reply:
Oh yes, they should be. I asked Bernardin to copy pattern of `InlayHintsEnabled()`, but that wasn't in camelCase by mistake. 

Bernardin, could you fix `InlayHintsEnabled()` to camelCase to if you're going to change `ParameterHintsEnabled()` to camelCase.